bag/datasets/bag/documents.py

```diff
@@ -355,6 +334,7 @@ def from_openbare_ruimte(o: models.OpenbareRuimte):
 
     d.naam = o.naam
     d.naam_nen = o.naam_nen
+    d.naam_no_ws = o.naam
```

Comment:
Is dit handiger of juist verwarrend?

`d.naam = d.naam_no_ws = o.naam`

Reply:
Ik zou het liever zo laten. Zie bijv :

    doc.straatnaam = openbare_ruimte.naam
    doc.straatnaam_no_ws = openbare_ruimte.naam
    doc.straatnaam_nen = openbare_ruimte.naam_nen
    doc.straatnaam_keyword = openbare_ruimte.naam
    doc.straatnaam_nen_keyword = openbare_ruimte.naam_nen

Daar word naam drie keer gebruikt en naam_nen twee keer